engine/Engine_Phonotype.sc

```diff
@@ -32,6 +32,13 @@ Engine_Phonotype : CroneEngine {
 			};
 
 		};
+
+		// does this auto wire things together? where should the OSCdef live?
+		OSCdef(\scCrowOut, { |msg, time, addr, recvPort|
+			msg.postln;
```

Comment:
this should call `luaOscAddr.sendMsg("/crow/out", msg[1].asInt, msg[2].asFloat, msg[3].asFloat);` i _think_